tests/monday_types_tests.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from unittest.mock import patch
+from nose.tools import ok_, eq_, raises
+from schematics.common import NONEMPTY
+
+from moncli import client, entities as en
+from moncli.entities import column_value as cv
+from moncli.enums import ColumnType
+from moncli import types as t
+
+
+def test_should_succeed_when_to_native_returns_an_int_or_float_when_passed_a_numbervalue_value_with_api_data():
+    pass
```

Comment:
I see this is causing you trouble.  Try this.  It will work.  It should also work for the other tests that you are doing.

    # Arrange
    number_type = t.NumberType(id='number_1')
    column_value = cv.create_column_value(ColumnType.numbers, id='number_1', value=json.dumps(1))

    # Act
    int_value = number_type.to_native(column_value)

    # Assert 
    eq_(int_value, 1)